environments/auth/package.json

```diff
@@ -3,6 +3,8 @@
   "name": "environments-auth",
   "version": "0.0.1",
   "scripts": {
-    "pull": "../pull-environments.sh"
+    "pull": "../pull-environments.sh -r us-east-1",
+    "pull:phone": "../pull-environments.sh -r us-east-1 -i \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\"",
```

Comment:
Would it make sense to reuse the `pull` script for the `phone` and `non-phone` scripts? So something like:
```
"pull-phone": "yarn pull -i \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\"",
```

Reply:
Ooh, that's a great suggestion! Let me try that.

Reply:
Works like a charm! [07ac032](https://github.com/aws-amplify/amplify-ui/pull/2609/commits/07ac03231ab8844265a48e15b167beb849c427d1)